wix-integration/README.md

```diff
@@ -0,0 +1,77 @@
+# Wix integration with GraphApi
+
+To show public data from GraphApi inside the Wix page, try the following steps:
```

Comment:
I wouldn't write `try` - this sounds like we are not sure ;)
I think we can be confident and write `perform the following steps`